controllers/skills.js

```diff
@@ -0,0 +1,70 @@
+const skills = require("../models/skills");
+const logger = require("../utils/logger");
+
+/**
+ * Award a skill
+ *
+ * @param req {Object} - Express request object
+ * @param req.body {Object} - User object
+ * @param res {Object} - Express response object
+ */
+async function awardSkill(req, res) {
+  try {
+    const { username } = req.params;
+    const addedData = await skills.awardSkill(req.body, username);
+    return res.json({
+      message: "Added data successfully!",
```

Comment:
Can we return a better response message?

Reply:
as in more descriptive? or something else entirely?

Reply:
A message that should be based on the acknowledgment of the action this api performs
Currently it's very generic `Added data successfully!`. It's ambiguous that what data is actually added